flocx_ui/static/dashboard/project/flocx/contract-list/contract-list.html

```diff
@@ -0,0 +1,43 @@
+<div ng-controller="ContractListController as ctrl">
+  <div class="col-lg-10">
+    <h3 style="margin: 24px 0;">Contracts</h3>
+    <div class="btn-group" role="group">
+      <button
+        type="button"
+        class="btn"
+        ng-class="{ 'btn-default': ctrl.pane !== 'active', 'btn-primary': ctrl.pane === 'active' }"
```

Comment:
Same comment. Let us avoid using 'magic vales'. 
[https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad](url)